pipe.go

```diff
@@ -159,6 +184,10 @@ func DialPipe(path string, timeout *time.Duration) (net.Conn, error) {
 		}
 		err = waitNamedPipe(path, ms)
 		if err != nil {
+			if shouldRetry(err) {
```

Comment:
I don't believe this is needed here.  Not only that, I don't think this error can occur.